docs/user-guide.md

```diff
@@ -474,13 +613,25 @@ TODO
 from graphframes.examples import Graphs
 g = Graphs(sqlContext).friends()  # Get example graph
 
-TODO
+# Run PageRank until convergence to tolerance "tol".
+results = g.pageRank(resetProbability=0.15, tol=0.01)
+# Display resulting pageranks and final edge weights
+results.vertices.select("id", "pagerank").show()
+results.edges.select("src", "dst", "weight").show()
+
+# Run PageRank for a fixed number of iterations.
+results2 = g.pageRank(resetProbability=0.15, numIter=10)
+
+# Run PageRank personalized for vertex "a"
+results3 = g.pageRank(resetProbability=0.15, numIter=10, sourceId="a")
 {% endhighlight %}
 </div>
 
 </div>
 
-## SVD++
+## Shortest paths
+
+Computes shortest paths to the given set of landmark vertices.
```

Comment:
It would be useful to explain how to reference a landmark (by vertex ID) in the user guide.